src/components/VSelectField/VSelectField.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div
+    class="relative m-0.5px box-content block w-fit overflow-hidden rounded-sm border text-sm focus-within:m-0 focus-within:border-1.5 focus-within:border-pink hover:border-dark-charcoal focus-within:hover:border-pink"
+    :class="variant === 'borderless' ? 'border-tx' : 'border-dark-charcoal-20'"
+  >
+    <div class="pointer-events-none absolute inset-y-0 my-auto h-fit start-2">
+      <slot name="start" />
+    </div>
+    <div class="pointer-events-none absolute inset-y-0 my-auto h-fit end-2">
+      <VIcon :icon-path="caretDownIcon" />
+    </div>
+    <select
+      :id="fieldId"
+      v-model="modelMedium"
+      class="flex h-[calc(theme(spacing.10)_-_2_*_theme(borderWidth.DEFAULT))] w-full appearance-none bg-white pe-10 truncate"
+      :class="hasStartContent ? 'ps-10' : 'ps-2'"
+      :name="fieldName"
+      v-bind="$attrs"
+      :aria-label="labelText"
+      v-on="$listeners"
+    >
+      <option v-if="blankText" disabled value="">{{ blankText }}</option>
+      <option v-for="choice in choices" :key="choice.key" :value="choice.key">
+        {{ choice.text }}
+      </option>
+    </select>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed, PropType } from '@nuxtjs/composition-api'
+
+import { defineEvent } from '~/types/emits'
+
+import VIcon from '~/components/VIcon/VIcon.vue'
+
+import { ProperlyExtractPropTypes } from '../../types/prop-extraction'
+
+import caretDownIcon from '~/assets/icons/caret-down.svg'
```

Comment:
```suggestion
import { defineEvent } from '~/types/emits'
import { ProperlyExtractPropTypes } from '~/types/prop-extraction'

import VIcon from '~/components/VIcon/VIcon.vue'

import caretDownIcon from '~/assets/icons/caret-down.svg'
```

Reply:
Usually, Webstorm adds imports as `../../`, but the standard way to import in the repo is using `~` alias. When you change `../..` to `~`, the linter starts complaining about the imports order.

Reply:
Is there a way for linting to catch this? WebStorm keeps using the relative import style and there doesn't seem to be any preference option to change this.

Reply:
Fixed in 205bfe5.